#action dispatch messages
int32 packed_valid
string drone_name
string drone_mode
int32 waypoint_valid
float32 x
float32 y
float32 z
int8 action_id
